### ISSUE TITLE
How to let image in full screen  

### ISSUE THREAD

**Issue by wayne811** [RubenGees/Introduction#15]
As Caption. 

new FullscreenStyle() is not work. Please help and thanks.

**Comment by rubengees** [comment]
Could you provide some additional information:

- Android Version? Device?
- How does your code look like?
- How does the result look like?

**Comment by wayne811** [comment]
![untitled](https://user-images.githubusercontent.com/18750447/27618893-6fe44080-5bf1-11e7-892b-a84bcdd268fc.png)

I would like to remove the title and description textView.
And let the imageview in full screen but i dont know how to do it. 

My Code:
```java
public class MyActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        new IntroductionBuilder(this).withSlides(generateSlides()).withStyle(new FullscreenStyle()).introduceMyself();
    }

    private List<Slide> generateSlides() {
        List<Slide> result = new ArrayList<>();

        result.add(new Slide()
                .withColorResource(R.color.colorAccent)
                .withImage(R.drawable.a)
        );

        result.add(new Slide()
                .withColorResource(R.color.colorPrimary)
                .withImage(R.drawable.b)
        );

        result.add(new Slide()
                .withColorResource(R.color.colorPrimaryDark)
                .withImage(R.drawable.c)
        );


        return result;
    }
}
```

**Comment by rubengees** [comment]
Ah, I see. The `FullscreenStyle` only disables the transparent navigation and status bar.
It is currently not possible to make the image occupy the entire space, i'll work later on that.

**Comment by kantbinh** [comment]
hi @rubengees 
Can you please update for this issue ?

**Comment by rubengees** [comment]
@wayne811 @nguyencongbinh Sorry, totally forgot about this issue!

This is now available as of version [2.0.0](https://github.com/rubengees/introduction/releases/tag/2.0.0).
